internal/resources/configmap_test.go

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Maybe it would be better to adopt the more common convention of SPDX-License-Identifier (a single comment at the top of each source file):

```// SPDX-License-Identifier: Apache-2.0```

Reply:
Sounds reasonable, but let's dedicate a separate PR (or more) to that effort.